src/main/java/teammates/ui/controller/InstructorCourseEditPageData.java

```diff
@@ -69,24 +69,16 @@ public InstructorCourseEditPageData(AccountAttributes account, CourseAttributes
         return panelList;
     }
 
-    private void createButtons(InstructorAttributes currentInstructor) {
-        boolean isDisabled = !currentInstructor.isAllowedForPrivilege(
+    private void createCourseRelatedButtons(InstructorAttributes currentInstructor) {
+        boolean isEditDeleteButtonDisabled = !currentInstructor.isAllowedForPrivilege(
                                                     Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_COURSE);
         
-        String editCourseButtonContent = "<span class=\"glyphicon glyphicon-pencil\"></span> Edit";
-        editCourseButton = createButton(editCourseButtonContent, "btn btn-primary btn-xs", "courseEditLink",
-                                        "javascript:;", Const.Tooltips.COURSE_INFO_EDIT, null, isDisabled);
-        
-        String content = "<span class=\"glyphicon glyphicon-trash\"></span>Delete";
-        String onClick = "return toggleDeleteCourseConfirmation('" + course.getId() + "');";
-        deleteCourseButton = createButton(content, "btn btn-primary btn-xs", "courseDeleteLink",
-                                          getInstructorCourseDeleteLink(course.getId(), false),
-                                          Const.Tooltips.COURSE_DELETE, onClick, isDisabled);
-        
-        isDisabled = !currentInstructor.isAllowedForPrivilege(
+        editCourseButton = createEditCourseButton(isEditDeleteButtonDisabled);
+        deleteCourseButton = createDeleteCourseButton(isEditDeleteButtonDisabled);
+
+        boolean isAddButtonDisabled = !currentInstructor.isAllowedForPrivilege(
```

Comment:
`isAddInstructorButtonDisabled` because `isAddButtonDisabled` doesn't tell what object to add.